README.md

```diff
@@ -21,7 +28,7 @@ Install python 3.5 or greater. This example uses 3.6.
 
 ```
 [auth_microservice] $ git clone https://github.com/heliumdatacommons/auth_microservice.git
-[auth_microservice] $ cd auth_microservice/
+[auth_microservice] $ cd auth_microservice/src/microservice/
```

Comment:
@BenjaminHCCarr after the previous PR was merged, please changes instances of `src` to `example` and I will merge this one